buildci.sh

```diff
@@ -23,44 +23,55 @@ function mvnp() {
         stdbuf -o0 sed -uE 's/^\[INFO\] Building (.*[^ ])[ ]+\[([0-9]+\/[0-9]+)\]$/\2| \1/' | # prefix project name with progress
         stdbuf -o0 sed -e :a -e 's/^.\{1,6\}|/ &/;ta' & # right align progress with padding
     local pid=$!
-    prevent_timeout $pid &
-    wait $pid
+    prevent_timeout ${pid} &
+    wait ${pid}
 }
 
 COMMITS=${1:-"master...HEAD"}
 
 # Determine if this is a new addon -> Perform tests + integration tests and all SAT checks with increased warning level
-CHANGED_DIR=`git diff --dirstat=files,0 $COMMITS bundles/ | sed 's/^[ 0-9.]\+% bundles\///g' | grep -o -P "^([^/]*)" | uniq`
+CHANGED_BUNDLE_DIR=`git diff --dirstat=files,0 ${COMMITS} bundles/ | sed 's/^[ 0-9.]\+% bundles\///g' | grep -o -P "^([^/]*)" | uniq`
+CHANGED_ITEST_DIR=`git diff --dirstat=files,0 ${COMMITS} itests/ | sed 's/^[ 0-9.]\+% itests\///g' | sed 's/\.tests\///g' | uniq`
```

Comment:
Took a few moments before I understood the black magic with the `.tests` string. Could you maybe add a comment either here or down at line 59?